app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Here is the list of all users</h1>
\ No newline at end of file
+<%= content_tag(:div, class: "blog-container") do %>
+    <%= content_tag(:h1, "Blog app", class: "blog-heading") %>
+    <% @users.each do |user| %>
+      <%= render partial: "users/user", locals: { user: user } %>
```

Comment:
- [ ] This is super seeing you went over and made use of partials in your views! Good job and keep this up :clap: